### ISSUE TITLE
[Bug]: No longer working on WotLK classic in TOGC patch

### ISSUE THREAD

**Issue by synze** [wardz/Diminish#39]
### What happened?

Addon does not appear to throw bugs (except when using the test functionality) but simply does not add DRs to nameplates or frames anymore.

### Is there any errors?

```shell
1x Diminish_Options/testmode.lua:160: attempt to call method 'UseRaidStylePartyFrames' (a nil value)
[string "@Diminish_Options/testmode.lua"]:160: in function `ToggleArenaAndPartyFrames'
[string "@Diminish_Options/testmode.lua"]:376: in function `Test'
[string "@Diminish_Options/main.lua"]:274: in function <Diminish_Options/main.lua:264>

Locals:
self = Frame {
 ToggleArenaAndPartyFrames = <function> defined @Diminish_Options/testmode.lua:108
 IsTesting = <function> defined @Diminish_Options/testmode.lua:100
 pool = <table> {
 }
 IsAnchoring = <function> defined @Diminish_Options/testmode.lua:104
 IsTestingOrAnchoring = <function> defined @Diminish_Options/testmode.lua:96
 ShowAnchors = <function> defined @Diminish_Options/testmode.lua:327
 Test = <function> defined @Diminish_Options/testmode.lua:356
 0 = <userdata>
 HideAnchors = <function> defined @Diminish_Options/testmode.lua:218
 CreateDummyAnchor = <function> defined @Diminish_Options/testmode.lua:235
}
state = true
forceHide = nil
settings = <table> {
 nameplate = <table> {
 }
 player = <table> {
 }
 focus = <table> {
 }
 target = <table> {
 }
 arena = <table> {
 }
 party = <table> {
 }
}
showFlag = true
useCompact = false
(*temporary) = nil
(*temporary) = <table> {
 IsEditModeActive = <function> defined @SharedXML/Stubs.lua:7
}
(*temporary) = "attempt to call method 'UseRaidStylePartyFrames' (a nil value)"
isTesting = false
isAnchoring = false
L = <table> {
 WATCHFRIENDLY_TOOLTIP = "Toggle diminishing returns tracking for friendly players. (Cast by enemy onto friendly)

Enabling this may drastically increase memory usage in large Battlegrounds."
 ATTACH_PERSONAL_NAMEPLATE = "Anchor To Personal Nameplate"
 DEFAULT = "Default"
 SELECTBORDER_TOOLTIP = "Choose between different border textures for the icons. You may also skin the icons using Masque."
 PARTY = "Party"
 ANCHORDRAG = "%s
Grows: %s"
 RESETPOS_TOOLTIP = "Reset icon positions for this unitframe back to default values."
 COPY_TOOLTIP = "Copies settings from one existing profile into the currently active profile."
 NAMEPLATE = "Nameplates"
 RESETPROFILE = "Reset Profile"
 HEADER_CATEGORIES = "Enabled Categories"
 COMPACTFRAMES_ERROR = "Unable to test raid frames while not in a group."
 GROW_RIGHT = "Right"
 PROFILEACTIVE = "That profile is already active."
 COMBATLOCKDOWN_ERROR = "Must leave combat or battleground before doing that."
 UNLOCK = "Toggle Moving Frames"
 TRACKNPCS_TOOLTIP = "Enable diminishing returns tracking for mobs and player pets. (Target/Focus/Nameplates)

Enabling this may drastically increase memory usage in large battlefields. In classic WoW this will only enable pet tracking."
 DELETE_TOOLTIP = "Delete an existing profile. Deletes and resets to default if the profile chosen is the currently active one."
 TEST = "Toggle Test Mode"
 TIMERTEXT_TOOLTIP = "Toggles the cooldown countdown text for all timers.

(Esc -> Interface -> ActionBars -> Show Numbers for Cooldowns)"
 ATTACH_PERSONAL_NAMEPLATE_TOOLTIP = "Attaches DR icons to your own personal resource display nameplate instead of the player frame."
 TIMERSWIPE = "Show Swipe for Cooldowns"
 TARGETTIP = "Target/focus yourself to see all frames."
 HEADER_ICONS = "Icons"
 COLORBLIND_TOOLTIP = "Show DR indicator numbers next to icons. 1 = first DR, 2 = second DR, 3 = last DR. May not work with some Masque skins.
(Color Blind Mode)"
 ICONSIZE_TOOLTIP = "Set the size of the icons."
 TIMERTEXTSIZE_TOOLTIP = "Set the font size of the timer countdown text."
 SELECTPROFILE = "Select Profile"
 NAMEPLATES = "Nameplates"
 ZONE_SCENARIO = "Scenario"
 TIMEREDGE_TOOLTIP = "Sets whether a bright line should be drawn on the moving edge of the cooldown animation effect."
 NEWPROFILE = "Create New Profile"
 HEADER_UNITFRAME = "Configuration for %s frame tracking."
 ENABLED = "Enabled"
 ICONPADDING_TOOLTIP = "Set the padding between all active icons."
 TARGET = "Target"
 ZONE_ARENA = "Arena"
 ZONES_TOOLTIP = "Enable tracking for this zone."
 ZONE_RAIDS = "Raids"
 ZONE_OUTDOORS = "World"
 ZONE_DUNGEONS = "Dungeons"
 PLAYER = "Player"
 NEWPROFILE_TOOLTIP = "Create a new profile with current active settings as a starting point."
 COPY = "Copy"
 RESETPOS = "Reset Position"
 TIMEREDGE = "Show Edge Effect on Swipe"
 ZONE_BATTLEGROUNDS = "Battlegrounds & Brawls"
```

### Game Version

Wrath of the Lich King

### Game Language

English

### Does the issue still occur when all other addons are temporarily disabled?

Not sure.

### Any additional info?

I know the addon is no longer maintained. If this request is unwelcome, please feel free to ignore it, and know I very much appreciate all your work on it over the years. Cheers :)

**Comment by wardz** [comment]
Thanks, should hopefully be fixed in v2.6.3

**Comment by synze** [comment]
You're amazing. Thank you so much!
